android/app/src/main/java/org/fossasia/openevent/core/schedule/DayScheduleFragment.java

```diff
@@ -238,10 +196,8 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
     @Override
     public boolean onQueryTextChange(String query) {
         searchText = query;
-
-        dayScheduleAdapter.filter(searchText);
+        filterByQuery(searchText);
```

Comment:
@srv-twry Instead of dayScheduleAdapter.filter function, I created another function filterByQuery to use the ViewModel instead.

Reply:
Now there is filterByQuery and filterByTracks.